ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/viewmodel/CustomerCenterViewModel.kt

```diff
@@ -1000,6 +1013,14 @@ internal class CustomerCenterViewModelImpl(
                     CustomerCenterManagementOption.CustomUrl(it.toUri())
                 }
 
+            CustomerCenterConfigData.HelpPath.PathType.CUSTOM_ACTION ->
+                path.actionIdentifier?.let { actionIdentifier ->
+                    CustomerCenterManagementOption.CustomAction(
+                        actionIdentifier = actionIdentifier,
+                        purchaseIdentifier = null, // This will be set appropriately when called from the UI
```

Comment:
I am confused about this comment

Reply:
Actually, at the beginning it thought purchaseInformation wasn't available here, but I can add it to `notifyListenersForManagementOptionSelected`. I'll update this